apps/sv/frontend/src/components/votes/actions/CreateUnallocatedUnclaimedActivityRecord.tsx

```diff
@@ -0,0 +1,155 @@
+  chooseAction: (action: ActionRequiringConfirmation) => void;
+  action?: ActionFromForm;
+  effectivity: Dayjs;
+}> = ({ chooseAction, action, effectivity }) => {
+  const existing = asCreateUnallocatedUnclaimedActivityRecord(action);
+
+  const [beneficiary, setBeneficiary] = useState(existing?.beneficiary ?? '');
+  const [amount, setAmount] = useState(existing?.amount ?? '');
+  const [reason, setReason] = useState(existing?.reason ?? '');
+  const [expiresAt, setExpiresAt] = useState<Dayjs>(dayjs());
+  const [amountError, setAmountError] = useState<string | null>(null);
+
+  useEffect(() => {
+    if (existing?.expiresAt) {
+      setExpiresAt(dayjs(existing.expiresAt));
+    } else {
+      setExpiresAt(effectivity.add(1, 'day'));
+    }
+  }, [existing?.expiresAt, effectivity]);
+
+  const updateAction = useCallback(
+    (beneficiary: string, amount: string, reason: string, expiresAt: Dayjs) => {
+      chooseAction({
+        tag: 'ARC_DsoRules',
+        value: {
+          dsoAction: {
+            tag: 'SRARC_CreateUnallocatedUnclaimedActivityRecord',
+            value: {
+              beneficiary,
+              amount,
+              reason,
+              expiresAt: expiresAt.toISOString(),
+            },
+          },
+        },
+      });
+    },
+    [chooseAction]
+  );
+
+  useEffect(() => {
+    if (beneficiary && amount && reason && expiresAt && !amountError) {
+      updateAction(beneficiary, amount, reason, expiresAt);
+    }
+  }, [beneficiary, amount, reason, expiresAt, amountError, updateAction]);
+
+  return (
+    <Stack direction="column" mb={4} spacing={1}>
+      <Typography variant="h6">Beneficiary</Typography>
+      <FormControl fullWidth>
+        <TextField
+          id="create-beneficiary"
+          value={beneficiary}
+          onChange={e => setBeneficiary(e.target.value)}
+        />
+      </FormControl>
+
+      <Typography variant="h6">Amount</Typography>
+      <FormControl fullWidth>
+        <TextField
+          id="create-amount"
+          type="number"
+          value={amount}
+          inputProps={{ 'data-testid': 'create-amount' }}
+          onChange={e => {
+            const newValue = e.target.value;
+            setAmount(newValue);
+
+            try {
+              const decimal = new Decimal(newValue);
+              if (decimal.lte(0)) {
+                setAmountError('Amount must be a positive number');
+              } else {
+                setAmountError(null);
+              }
+            } catch {
+              setAmountError('Amount must be a positive number');
+            }
+          }}
+          error={!!amountError}
+          helperText={amountError}
+        />
+      </FormControl>
+
+      <Typography variant="h6">Reason</Typography>
+      <FormControl fullWidth>
+        <TextField id="create-reason" value={reason} onChange={e => setReason(e.target.value)} />
+      </FormControl>
+
+      <Typography variant="h6">Can be claimed until</Typography>
```

Comment:
```suggestion
      <Typography variant="h6">Can be minted until</Typography>
```

@waynecollier-da is that the right phrasing?

Reply:
this one still seems unresolved, let's hold off on merging until Wayne comments

Reply:
Ad discussed with @waynecollier-da we will use 'Must Mint Before'.